dap_aria_mapping/utils/patents.py

```diff
@@ -0,0 +1,30 @@
+"""
+Utils related to collecting patents data
+"""
+from typing import List
+
+
+def chunk(lst: List[str], n: int) -> List[List[str]]:
+    """Chunk a list of strings into n lists of strings.
+
+    Args:
+        lst (List[str]): List of strings to be chunked.
+        n (int): Number of elements per chunk.
+
+    Returns:
+        List[List[str]]: List of lists of chunked strings.
+    """
+    return [lst[i::n] for i in range(n)]
```

Comment:
Does chunking by number of chunks make sense here? If the number of results is really large, then you could end up with really big chunks. OTOH you could chunk by number of items per chunk.